assets/charts/components/istio-operator/Chart.yaml

```diff
@@ -1,11 +1,12 @@
 apiVersion: v2
 name: istio-operator
-version: 1.6.0
+version: 1.7.0
+tillerVersion: ">=2.7.2"
```

Comment:
Did they really add that now? Isn't tiller deprecated since long time?

Reply:
It seems that they support v2 as well. Like if you see new changes in the values file, they have added a workaround for CRD when installed using helm v2.